docs/compute-element/htcondor-ce-overview.md

```diff
@@ -132,17 +131,14 @@ installation, there are two main ways to customize your CE (if you decide any cu
 How Security Works
 ------------------
 
-In the OSG, security depends on a PKI infrastructure involving Certificate Authorities (CAs) where CAs sign and issue
-certificates.
-When these clients and hosts wish to communicate with each other, the identities of each party is confirmed by
-cross-checking their certificates with the signing CA and establishing trust.
-
-In its default configuration, HTCondor-CE uses GSI-based authentication and authorization to verify the certificate
-chain, which will work with [LCMAPS VOMS authentication](../security/lcmaps-voms-authentication.md).
-Additionally, it can be reconfigured to provide alternate authentication mechanisms such as Kerberos, SSL, shared
-secret, or even IP-based authentication.
-More information about authorization methods can be found
-[here](http://research.cs.wisc.edu/htcondor/manual/v8.6/3_8Security.html#SECTION00483000000000000000).
+In a distributed environment such as the OSG, communication is secured between various parties using a combination of
```

Comment:
I'd just keep it as "In the OSG" otherwise the sentence implies this is how it works in all distributed environments...

Reply:
Done